src/components/CheckEditor/FormComponents/RequestBodyContentType.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useId } from 'react';
+import { FieldPath, useFormContext } from 'react-hook-form';
+import { Field, Input } from '@grafana/ui';
+
+import { CheckFormValues } from 'types';
+
+interface RequestBodyContentTypeProps {
+  disabled?: boolean;
+  name: FieldPath<CheckFormValues>;
+}
+
+export const RequestBodyContentType = ({ disabled, name }: RequestBodyContentTypeProps) => {
+  const { register } = useFormContext<CheckFormValues>();
+  const id = useId().replace(/:/g, '_');
```

Comment:
This is done in several places, should we move it to a common function/hook?

Reply:
Done, added `useDOMId` with a comment explaining its purpose.